contents/javascript/Javascript-framework-React.md

```diff
@@ -0,0 +1,134 @@
+<frontmatter>
+  title: React
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# React
+
+Author: [Aadyaa Maddi](https://github.com/amad-person)
+
+### What is React?
+
+**React** is an open-source JavaScript <tooltip content="React only handles the view component of your application. It is not an MVC framework!">library</tooltip> for building composable user interfaces. It was developed by Facebook and released to the world in 2013, and is now used in a huge number of popular applications, including Facebook and Instagram.
+
+<box type="tip">
+    If you have the <a href="https://github.com/facebook/react-devtools">React DevTools</a> extension installed in your browser, you can inspect the webpage of either of these applications and observe that a lot of React components are being used by them! 
+</box>
+
+React abstracts away the [DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction) from you, giving you a simpler programming model and better performance. It can also render on the server using Node, and it can power native mobile applications using [React Native](https://facebook.github.io/react-native/).
```

Comment:
I think it would be good to start a new sentence/paragraph and separate React native from React.

So, you can say something like `A slight variation of React is React Native...used for building native apps for Android and iOS...`
Also, React Native is a framework (vs React, which is a library)

Reply:
> I think it would be good to start a new sentence/paragraph and separate React native from React.

Although there is merit in elaborating on React Native, I'm wondering if this would take the focus away from the main topic (React).

Reply:
Fair enough, you can either mention it as an aside (in an info box) or you can leave the structure the way it is. However, I think you should at least mention something along the lines of `... using React Native, which is an extension of React.`

It's your call